src/SFA.DAS.EmployerIncentives.Api/Authentication/AzureAdScopeClaimTransformation.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authentication
+{
+    public class AzureAdScopeClaimTransformation : IClaimsTransformation
```

Comment:
As per previous comment, looks like this could be shared?

Reply:
Agreed